### PR TITLE
xray: Add version 1.0.0

### PR DESCRIPTION
**Pull request by lifansama**
Xray, Penetrates Everything. Also the best v2ray-core, with XTLS support. Fully compatible configuration.

### CODE REVIEW

File: bucket/xray.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+			"url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-32.zip",
+			"hash": "18E099E12B9DCEAAE546D840C2BEEE00A3BB2D47EB390B347CF33E3372E2F9B6"
+		}
+	},
+	"bin": [
+		"xray.exe"
+	],
+	"checkver": {
+		"github": "https://github.com/XTLS/Xray-core"
+	}
```
Comment (rasa) [inline bucket/xray.json:21]:
```suggestion
	},
	"autoupdate": {
		"64bit": {
			"url": "https://github.com/XTLS/Xray-core/releases/download/v$version/Xray-windows-64.zip"
		},
		"32bit": {
			"url": "https://github.com/XTLS/Xray-core/releases/download/v$version/Xray-windows-32.zip"
		}
	}
```

**Review by rasa — CHANGES_REQUESTED** [review]

File: bucket/xray.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+	"version": "1.0.0",
+	"description": "Xray, Penetrates Everything. Also the best v2ray-core, with XTLS support. Fully compatible configuration.",
+	"homepage": "https://github.com/XTLS/Xray-core",
+	"license": "MPL",
```
Comment (kidonng) [inline bucket/xray.json:5]:
```suggestion
	"license": "MPL-2.0",
```

File: bucket/xray.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+	"version": "1.0.0",
+	"description": "Xray, Penetrates Everything. Also the best v2ray-core, with XTLS support. Fully compatible configuration.",
```
Comment (kidonng) [inline bucket/xray.json:3]: Don't use the advertising words from the project itself. You may check out [an example](https://github.com/kidonng/sushi/blob/21e8c412c234324267ccdfc7f008296882a69387/bucket/xray.json#L3) of an appropriate description.

File: bucket/xray.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+			"url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-64.zip",
+			"hash": "9B525F9945AAC08A0752055CFD3EFA7779A8D86DC2D4BC1EDFA49B6300593CD1"
+		},
+		"32bit": {
+			"url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-32.zip",
+			"hash": "18E099E12B9DCEAAE546D840C2BEEE00A3BB2D47EB390B347CF33E3372E2F9B6"
+		}
+	},
+	"bin": [
+		"xray.exe"
```
Comment (kidonng) [inline bucket/xray.json:17]: Remove the array.

File: bucket/xray.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+	],
+	"checkver": {
+		"github": "https://github.com/XTLS/Xray-core"
```
Comment (kidonng) [inline bucket/xray.json:20]: Just use `"checkver": "github"`

**Review by kidonng — APPROVED** [review]

File: bucket/xray.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+	}
+}
```
Comment (earnestma) [inline bucket/xray.json:22]: tabs -> spaces

```suggestion
{
    "version": "1.0.0",
    "description": "Xray, Penetrates Everything. Also the best v2ray-core, with XTLS support. Fully compatible configuration.",
    "homepage": "https://github.com/XTLS/Xray-core",
    "license": "MPL-2.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-64.zip",
            "hash": "9B525F9945AAC08A0752055CFD3EFA7779A8D86DC2D4BC1EDFA49B6300593CD1"
        },
        "32bit": {
            "url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-32.zip",
            "hash": "18E099E12B9DCEAAE546D840C2BEEE00A3BB2D47EB390B347CF33E3372E2F9B6"
        }
    },
    "bin": [
        "xray.exe"
    ],
    "checkver": {
        "github": "https://github.com/XTLS/Xray-core"
    }
}
```

**Review by earnestma — CHANGES_REQUESTED** [review]

File: bucket/xray.json (hunk 3 above)
Comment (earnestma) [inline bucket/xray.json:3]: Description

```suggestion
{
    "version": "1.0.0",
    "description": "Network tools for the XTLS protocol",
```

File: bucket/xray.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+	"architecture": {
+		"64bit": {
+			"url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-64.zip",
+			"hash": "9B525F9945AAC08A0752055CFD3EFA7779A8D86DC2D4BC1EDFA49B6300593CD1"
+		},
+		"32bit": {
+			"url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-32.zip",
+			"hash": "18E099E12B9DCEAAE546D840C2BEEE00A3BB2D47EB390B347CF33E3372E2F9B6"
+		}
+	},
```
Comment (earnestma) [inline bucket/xray.json:15]: tabs -> spaces

```suggestion
    "homepage": "https://github.com/XTLS/Xray-core",
    "license": "MPL-2.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-64.zip",
            "hash": "9B525F9945AAC08A0752055CFD3EFA7779A8D86DC2D4BC1EDFA49B6300593CD1"
        },
        "32bit": {
            "url": "https://github.com/XTLS/Xray-core/releases/download/v1.0.0/Xray-windows-32.zip",
            "hash": "18E099E12B9DCEAAE546D840C2BEEE00A3BB2D47EB390B347CF33E3372E2F9B6"
        }
    },
```

File: bucket/xray.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+	"bin": [
+		"xray.exe"
+	],
```
Comment (earnestma) [inline bucket/xray.json:18]:
```suggestion
    "bin": "xray.exe",
```

File: bucket/xray.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+	"checkver": {
+		"github": "https://github.com/XTLS/Xray-core"
+	}
```
Comment (earnestma) [inline bucket/xray.json:21]:
```suggestion
    "checkver": "github",
```

**Review by earnestma — CHANGES_REQUESTED** [review]

**Pull request closed without merging**